test/specs/main.js

```diff
@@ -25,23 +25,34 @@ const testCase = function (err, appPaths) {
     assert.fail('The packaging process is failed!');
   }
   const appPath = path.resolve('.', appPaths[0]);
-  const resourcePath = path.resolve(appPath, `${APP_NAME}.app/Contents/Resources`);
-  const resources = fs.readdirSync(resourcePath);
-  const languageFolderFilter = new RegExp(`\.(lproj)$`);
-  const languageFolders = resources.filter((folder) => languageFolderFilter.test(folder));
-
-  languageFolders.forEach((languageFolder) => {
-    const language = languageFolder.split('.')[0];
-
-    if (!LANGUAGES.includes(language)) {
-      assert.fail(`This language, "${language}", shouldn't be in the app.`);
-    }
-  });
+  if (appPaths[0].includes('darwin') || appPaths[0].includes('mas')) {
```

Comment:
@fcastilloec do you think we could get rid of this if conditions to check the platform and make the test more generic with help of some functions to differentiate some platform differences? Besides that, I guess it's possible to run the test for `win32` platform even with current test code?

Reply:
Yes, this can be improved by setting some function depending on platform.

We can test for `win32` the problem is that we need to install `wine`. Which can be done with `homebrew`. Otherwise we can't pack for windows. Alternatively, you could just setup [AppVeyor](http://appveyor.com) to test for windows there.
This also reminds me that since we could test for all OSs from a single one (if `wine` is installed), why not use the default Linux image in Travis, which is faster and has a lot more capacity than OSX